specification/2.0/Makefile

```diff
@@ -8,19 +8,18 @@ all: $(TARGETS)
 
 # Generate the Properties Reference section of the spec from JSON schema.
 # This uses 'npx' to minimally install the 'wetzel' tool, if needed.
-# While the schema used by wetzel is the locally installed version, it
-# generates inline links to the schema under $(SCHEMALINK), typically
-# the canonical schema in the github default branch.
+# When using embedded schemas, SCHEMALINK should be a relative path.
 
 WETZEL = npx wetzel
-SCHEMALINK = https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema
+SCHEMALINK = schema
+EMBEDSCHEMA = JsonSchemaReference.adoc
 
 # Base name of the generated properties reference file
 PROPREF = PropertiesReference
 # Schema files to leave out of $(PROPREF)
 IGNORESCHEMA = '["gltfchildofrootproperty.schema.json", "gltfid.schema.json", "gltfproperty.schema.json"]'
 $(PROPREF).adoc: $(wildcard schema/*.json)
-	$(WETZEL) -n -a=cqo -m=a -p "$(SCHEMALINK)" \
+	$(WETZEL) -n -a=cqo -m=a -p "$(SCHEMALINK)" -e "$(EMBEDSCHEMA)" \
```

Comment:
Is this where $(EMBEDSCHEMA) is generated?

Reply:
Yes. I suppose it would be more Makefile-style if we were to run wetzel twice for the two different targets.  In that case `-e` would be some kind of embed mode switch to indicate which of the two targets was being produced, and the output would always go through stdout.  Is it worth refactoring to get that behavior?

Reply:
I don't think it is - the two targets should be generated at the same time to remain consistent. Not a big deal either way.